tests/acceptance/client_admin/doi-test.js

```diff
@@ -137,4 +137,22 @@ module('Acceptance | client_admin | repository', function(hooks) {
     assert.equal(currentURL(), 'repositories/datacite.rph/dois/new');
     assert.dom('[doi-contributor]').includesText('DataCollector');
   });
+
+  test('visiting the Form and adding Alternate Identfier', async function(assert) {
+    await authenticateSession({
+      uid: 'datacite.rph',
```

Comment:
As per what @mfenner said this will need changing probably as the way auth testing has setup is different. Using a seperate test account.

Reply:
I think we can deal with that when those changes are made.